assets/src/edit-story/components/movable/cropStyle.js

```diff
@@ -0,0 +1,104 @@
+/**
+ * External dependencies
+ */
+import { createGlobalStyle } from 'styled-components';
+
+export const GlobalStyle = createGlobalStyle`
+  .crop-movable .moveable-control {
+    background: #222 !important;
+    border-radius: 0px !important;
+    border: none !important;
+    box-sizing: border-box !important;
+  }
+
+  .crop-movable .moveable-control.moveable-n,
+  .crop-movable .moveable-control.moveable-s,
+  .crop-movable .moveable-control.moveable-e,
+  .crop-movable .moveable-control.moveable-w {
+    border: 1px solid #fff !important;
+  }
+
+  .crop-movable .moveable-control.moveable-n,
+  .crop-movable .moveable-control.moveable-s {
+    width: 16px !important;
+    height: 4px !important;
+    margin-left: -8px !important;
+    margin-top: -2px !important;
+  }
+
+  .crop-movable .moveable-control.moveable-e,
+  .crop-movable .moveable-control.moveable-w {
+    width: 4px !important;
+    height: 16px !important;
+    margin-left: -2px !important;
+    margin-top: -8px !important;
+  }
+
+  .crop-movable .moveable-control.moveable-nw,
+  .crop-movable .moveable-control.moveable-ne,
+  .crop-movable .moveable-control.moveable-sw,
+  .crop-movable .moveable-control.moveable-se {
+    width: 16px !important;
+    height: 16px !important;
+    background: #fff !important;
+  }
+
+  .crop-movable .moveable-control.moveable-nw:after,
+  .crop-movable .moveable-control.moveable-ne:after,
+  .crop-movable .moveable-control.moveable-sw:after,
+  .crop-movable .moveable-control.moveable-se:after {
+    content: "" !important;
+    display: block !important;
+    position: absolute !important;
+    inset: 1px !important;
```

Comment:
Is this supported in most of the browsers? Perhaps it's not that important :)

Reply:
I assumed the CSS-in-JS library we use would expand this for non-supporting browsers. But if not, it'd be pretty easy to find them all and replace at any time.